src/functions/utils/messages.ts

```diff
@@ -0,0 +1,7 @@
+/* Copyright */
+export const messages = {
+  bonusMessage:
+    "Your total usable bonus is: {remainingTotalBudget}€\nAmount to be paid in cash: {remainingTotalPayable}€",
+  noUsernameDevMode: "No username provided in dev mode. Please provide the username in the command text.",
+  slackUserNotFound: "Your slack username was not found in the bonus excel, contact Mari",
```

Comment:
Let's add the the received username to this message so checking it will be easier if this message is received.